spotify-easyrpm

```diff
@@ -209,10 +209,9 @@ fi
 f_extract_snap() {
 echo "Extracting snap and preparing"
 unsquashfs -d "${V_SNAP_EXTRACT_DIR}"  "${V_SOURCES_DIR}"/"${V_HTTP_SNAP}" || f_error "Failed to extract snap"
-mkdir -p "${V_SOURCES_DIR}"/"${V_PKGNAME}"/usr/{share/applications,share/doc,bin}
+mkdir -p "${V_SOURCES_DIR}"/"${V_PKGNAME}"/usr/{share/applications,bin}
```

Comment:
@megamaced Please check he soundness of this change. I assume it is safe, but I tested the script before applying the change to this line.